cfme/tests/configure/test_config_diagnostics.py

```diff
@@ -68,3 +71,39 @@ def test_configuration_dropdown_roles_by_server(appliance, request):
     if BZ(1734393, forced_streams=["5.10"]).blocks:
         view.rolesbyservers.tree.select_item("SmartState Analysis")
     assert "available" in view.rolesbyservers.tree.currently_selected_role
+
+
+@test_requirements.general_ui
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1498090])
+@pytest.mark.parametrize('obj', ["Zone", "Region"])
+def test_diagnostics_server(appliance, obj):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/15h
+        testSteps:
+            1. Navigate to Configuration and go to Diagnostics accordion.
+            2. Click on Region, click on `Servers` tab
+                and select a server from the table and check the landing page.
+            3. Click on Zone, click on `Servers` tab
+                and select a server from the table and check the landing page.
+        expectedResults:
+            1.
+            2. Landing page must be `Diagnostics Server` summary page.
+            3. Landing page must be `Diagnostics Server` summary page.
+
+    Bugzilla:
+        1498090
+    """
+    context_obj = appliance.server.zone
```

Comment:
_Optional_ consider using a ternary here: 
```python,
context_obj = context_obj.region if obj == "Region" else appliance.server.zone
```

Reply:
Ay, I forgot something like this existed, I knew I was missing something! Thanks John! :)